mathics/builtin/stats.py

```diff
@@ -0,0 +1,1743 @@
+import sympy
+import sympy.stats
+from mathics.builtin.base import SympyFunction, Builtin
+from mathics.core.expression import Symbol, Complex, Integer, Real
+from mathics.core.convert import from_sympy, sympy_symbol_prefix
+
+
+
+class DistributionParam(object):
+    msg = ''
+
+    @staticmethod
+    def convert(value):
+        return
+
+    @staticmethod
+    def get(value): #gets value for check_too -- todo
+        return
+
+
+class RealParam(DistributionParam):
+    msg = 'realprm'
+
+    @staticmethod
+    def convert(value):
+        if isinstance(value, Complex):
+            return None
+        else:
+            # TODO
+            return value.to_sympy()
+
+    @staticmethod
+    def get(value): #gets value for check_too -- todo
+        return
+
+
+class PositiveRealParam(DistributionParam):
+    msg = 'posprm'
+
+    @staticmethod
+    def convert(value):
+        if isinstance(value, Symbol):
+            return sympy.Symbol(sympy_symbol_prefix + value.name, positive=True)
+        elif isinstance(value, Complex):
+            return None
+        else:
+            # TODO
+            return value.to_sympy()
+
+    @staticmethod
+    def get(value): #gets value for check_too -- todo
+        return
+
+
+class IntegerParam(DistributionParam):
+    msg = 'intprm'
+
+    @staticmethod
+    def convert(value):
+        if isinstance(value, Symbol):
+            # TODO
```

Comment:
I think it's okay to just pass it on (as a sympy symbol) but I didn't check that everything worked. That is, do we need `sympy.Symbol('foo', integer=True)`?